README.md

```diff
@@ -1,10 +1,20 @@
-# The DANDI project Website
-
-This is the website for the DANDI project.
+# DANDI project website
```

Comment:
I know it seems pedantic, but it's important not to bundle unrelated changes such as this into a given PR.

Reply:
Thanks Roni.  I have removed these changes.  Will submit as a separate pull request.